python/python/lance/dataset.py

```diff
@@ -120,6 +120,7 @@ def scanner(
         fragment_readahead: Optional[int] = None,
         scan_in_order: bool = True,
         fragments: Optional[Iterable[LanceFragment]] = None,
+        with_row_id: bool = False,
```

Comment:
Could we move this after the `*`? I think we want to do that going forward with all kwargs.

Reply:
Ok, sg.